### DISCUSSION TITLE
[Issue] 11 seconds Video length error while sharing from file manager

### DISCUSSION THREAD

**Discussion by RevealedSoulEven** [rodit/SnapMod#106] [Ideas & Feature Suggestions]
Yes I know that we can add videos from the camera roll but it's a quite disgusting feature because whenever selecting from the camera roll we have to select the newer videos and we cannot select the device folder and also if I need to select any older video that is something around 1 year or 
2 years ago then it is very hard to select that video or that particular photo from the camera roll so whenever I try to select that video from the gallery and send it to Snapchat than it shows the problem of 11 seconds that I am attaching with the screenshot so if you can remove that 11 second restriction or any way to select videos to be send directly from the file manager, then it would be great. thank you
![Screenshot_2022-05-11-15-36-31-738_com snapchat android](https://user-images.githubusercontent.com/49313374/167826290-7e2a1f9f-44d2-4d92-94a5-ae863dc65969.jpg)

**Comment by rodit** [comment]
Understandable - I will have a look at this in the next update.

**Reply by rodit** [reply]
v1.6.1 now bypasses video restrictions globally.

**Comment by RevealedSoulEven** [comment]
Thanks a lot. It's fixed. You guys are really next level modders. Really appreciate
